pkg/util/strings.go

```diff
@@ -37,17 +38,18 @@ func StringsClone(s string) string {
 
 // MergeSlicesParallel merge sorted slices in parallel
 // using the MergeSortedSlices function
-func MergeSlicesParallel(parallelism int, a ...[]string) []string {
+func MergeSlicesParallel(ctx context.Context, parallelism int, a ...[]string) ([]string, error) {
```

Comment:
Can you add a test case that returns an error due to context check?

Reply:
I was not sure how to add proper test for context cancellation case. 
 1. I could mock and add some wait time on iteration, and cancel context within this time. But adding wait time on tests? I'm not sure. 
 2. Cancel context before calling the function? but in this one we are not actually testing the real case.
 
There should be better way than the above two. I will add second one, but let me know if we have any other option to properly test it.

Reply:
> Cancel context before calling the function? but in this one we are not actually testing the real case.

I am thinking about before or in the middle of the call. If it is not easy to mock that, I think it is fine to skip.